api/soap/mantisconnect.wsdl

```diff
@@ -546,6 +550,16 @@
   <part name="issue" type="tns:IssueData" /></message>
 <message name="mc_issue_addResponse">
   <part name="return" type="xsd:integer" /></message>
+<message name="mc_issue_reportRequest">
+  <part name="username" type="xsd:string" />
+  <part name="password" type="xsd:string" />
+  <part name="issue" type="tns:IssueData" />
+  <part name="attach_note" type="xsd:string" />
```

Comment:
Why? The note is for attach, that is why it is "attach_note" - it is super-clear. "note" is less clear to me. Is it a note for report itself? For attached file? For submission process itself?